frontend/src/components/Dashboard/Dashboard.jsx

```
@@ -0,0 +1,222 @@
+              <div className="w-8 h-8 bg-blue-500 text-white rounded-full flex items-center justify-center">
+                <span>{getFirstLetter(user ? user.name : '')}</span>
+              </div>
+              <span className="ml-2 text-gray-700">{user ? user.name : 'Guest'}</span>
+              <svg
+                className="w-4 h-4 ml-1 text-gray-500"
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+              >
+                <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)